airflow/example_dags/example_neo4j.py

```diff
@@ -0,0 +1 @@
+../providers/neo4j/example_dags/example_neo4j.py
```

Comment:
In the airflow / example_dags directory, we only store examples that use operators that are part of the core.
http://apache-airflow-docs.s3-website.eu-central-1.amazonaws.com/docs/apache-airflow/latest/operators-and-hooks-ref.html

Reply:
Thanks, will remove it.

Reply:
Removed.